.gitignore

```diff
@@ -69,4 +69,16 @@ installation/versions-overrides.env
 .changelog/
 
 # direnv files
-.envrc
\ No newline at end of file
+.envrc
+
+.vitepress/cache/
+.vitepress/dist/
+.DS_Store
+node_modules/
+# Ignore folder docs/external-content to avoid including external content in the repo
+docs/external-content/**
+# ...but include all the _sidebar.ts files in docs/external-content
+#       --> !docs/external-content/**/ to remove the rule ignoring all subfolders
+!docs/external-content/**/
+#       --> !docs/external-content/**/_sidebar.ts to include all _sidebar.ts files in all subfolders
+!docs/external-content/**/_sidebar.ts
```

Comment:
If I'm correct, it's not working, because I see a lot of the `external-content/` files.